### DISCUSSION TITLE
Need help with Dialogic scaling!

### DISCUSSION THREAD

**Discussion by AidanTilanus** [dialogic-godot/dialogic#2206] [Q&A] [answered]
In my game i use dialogic but because my game resolution is just 427x240 the dialogue box is really big. And i dont how how to make it smaller.

reddit post: https://www.reddit.com/r/godot/comments/1ccwhkk/comment/l1815q0/?context=3

![liyipgbhknwc1](https://github.com/dialogic-godot/dialogic/assets/137188145/4cafe15d-2775-49b4-81b9-74bdb9f9a12d)
![1rdmnebfknwc1](https://github.com/dialogic-godot/dialogic/assets/137188145/6ff73266-3f96-432d-a203-4bce0cb280b9)

**Comment by Jowan-Spooner** [comment, marked as answer]
Create a style and set the box size to be smaller:
![grafik](https://github.com/dialogic-godot/dialogic/assets/42868150/1d3656a4-d75c-46cf-849a-f96578753cc1)
